jscomp/bsc/rescript_compiler_main.ml

```diff
@@ -106,6 +106,43 @@ let process_file sourcefile ?(kind ) ppf =
   in
   Config.uncurried := uncurried;
   res
+
+let reprint_source_file sourcefile = 
+  Js_config.no_builtin_ppx := true;
+  let uncurried = !Config.uncurried in
+  let kind = Ext_file_extensions.classify_input (Ext_filename.get_extension_maybe sourcefile) in
+  let sourcefile = set_abs_input_name sourcefile in
+  let res = match kind with 
+  | Res -> 
+    let parseResult =
+      Res_driver.parsingEngine.parseImplementation ~forPrinter:true ~filename:sourcefile
+    in
+    if parseResult.invalid then (
+      Res_diagnostics.printReport parseResult.diagnostics parseResult.source;
+      exit 1);
+    parseResult.parsetree 
+    |> Js_implementation.implementationForReprinting 
+    |> Res_printer.printImplementation ~width:80 ~comments:parseResult.comments 
```

Comment:
Looks like width is only specified in 2 places: here and in the playground.
Everywhere else it defaults to 100.

Wondering whether we should even have a width parameter, or have it always be an optional arg which is never called.

Reply:
Yeah that's a good point. I should change it to 100 regardless here or the results might not be 100% correct.